salad/features/steps-browser-browsers.feature

```diff
@@ -11,6 +11,6 @@ Feature: Ensuring that javascript works in all supported browsers
 
     Examples:
     | browser   |
-    | chrome    |
+    #| chrome    | chrome does not work on travis at the moment
```

Comment:
what do you suggest? is there a travis alternative? we also shouldn't build just because "it works on my computer" which it does with chrome